Config.uk

```diff
@@ -6,6 +6,8 @@ menuconfig LIBNGINX
 	select LIBPOSIX_PROCESS
 	select LIBPOSIX_USER
 	select LIBPOSIX_SYSINFO
+	select LIBPOSIX_SOCKET
```

Comment:
I've just checked and there aren't any undefined references **without** this line. The `posix-socket` library is selected anyway. This is just an observation, we can keep it as it is, for continuity, since the other `posix` libraries are also included here.

Reply:
I would keep it as it is a direct dependency.

Reply:
@mariasfiraiala , I suggest then we keep it, as @marcrittinghaus said.

Reply:
@marcrittinghaus , is this OK from your side? Could @mariasfiraiala approve this PR before you do the final approval?